proto/buf.md

```diff
@@ -0,0 +1,10 @@
+# Connect Conformance
```

Comment:
I think it would be quite helpful to link to github.com/connectrpc/conformance, similar to eliza here: https://github.com/connectrpc/examples-go/blob/68b9b412e2210ada02debad8d65c3b45e190e825/proto/buf.md?plain=1#L14

Reply:
Agreed. I was going to do that after this merges to main. I could add it now, but I'd have to link it to the v1 branch which would then be a stale link once it merges.

Reply:
How about we go ahead and merge v1 to main first? What are we waiting on for that?